kokoro/gcp_ubuntu/kokoro_build.sh

```diff
@@ -3,6 +3,15 @@
 # Fail on any error.
 set -e
 
+#Variables
+GCP_PROJECT         =$(gcloud config get-value project)
+GCP_PROJECT_ID      =$(gcloud config get-value project)
```

Comment:
Please enclose non-literal arguments in quotes (i.e. `"$(gcloud ...)") to avoid spaces leading to an unwanted interpretation by bash.